exasol_integration_test_docker_environment/lib/test_environment/shell_variables.py

```diff
@@ -12,29 +12,30 @@ def __init__(self, env: Dict[str, str]):
     @classmethod
     def from_test_environment_info(
             cls,
-            default_bridge_ip_address: str,
+            default_bridge_ip_address: Optional[str],
             test_environment_info: EnvironmentInfo,
     ) -> 'ShellVariables':
         """
         Create ShellVariables from the given test_container_name,
         default_bridge_ip_address and EnvironmentInfo.
         """
         info = test_environment_info
-        env = {
+        env : Dict[str, str] = {
             "NAME": info.name,
             "TYPE": info.type,
             "DATABASE_HOST": info.database_info.host,
-            "DATABASE_DB_PORT": info.database_info.ports.database,
-            "DATABASE_BUCKETFS_PORT": info.database_info.ports.bucketfs,
-            "DATABASE_SSH_PORT": info.database_info.ports.ssh,
+            "DATABASE_DB_PORT": str(info.database_info.ports.database) if info.database_info.ports.database is not None else "",
+            "DATABASE_BUCKETFS_PORT": str(info.database_info.ports.bucketfs) if  info.database_info.ports.bucketfs is not None else "",
+            "DATABASE_SSH_PORT": str(info.database_info.ports.ssh) if info.database_info.ports.ssh is not None else "",
         }
         if info.database_info.container_info is not None:
+            assert default_bridge_ip_address
             network_aliases = " ".join(info.database_info.container_info.network_aliases)
             env.update({
                 "DATABASE_CONTAINER_NAME": info.database_info.container_info.container_name,
                 "DATABASE_CONTAINER_NETWORK_ALIASES": f'"{network_aliases}"',
                 "DATABASE_CONTAINER_IP_ADDRESS": info.database_info.container_info.ip_address,
-                "DATABASE_CONTAINER_VOLUMNE_NAME": info.database_info.container_info.volume_name,
+                "DATABASE_CONTAINER_VOLUMNE_NAME": info.database_info.container_info.volume_name or "",
```

Comment:
Probably also assert it, because we expect it to be available

Reply:
Done